examples/terraform-aws-lambda-example/src/lambda.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambda"
+)
```

Comment:
The build failed `goimports` formatting on this file. This is because the third party import is grouped with the stdlib imports. The build should move beyond that step once you add a new line between the `lambda` import and `fmt`.

Reply:
@yorinasub17 updated, thanks